public/app/core/utils/UserProvider.tsx

```diff
@@ -78,6 +92,28 @@ export class UserProvider extends PureComponent<Props, State> {
     this.setState({ teams, loadingStates: { ...this.state.loadingStates, loadTeams: false } });
   };
 
+  loadOrgs = async () => {
+    this.setState({
+      loadingStates: { ...this.state.loadingStates, loadOrgs: true },
+    });
+    const orgs = await getBackendSrv().get('/api/user/orgs');
+    this.setState({ orgs, loadingStates: { ...this.state.loadingStates, loadOrgs: false } });
+  };
+
+  setUserOrg = async (org: UserOrg) => {
+    this.setState({
+      loadingStates: { ...this.state.loadingStates, updateUserOrg: true },
+    });
+    await getBackendSrv()
```

Comment:
Maybe try to move this async operation out of the Component into a Thunk, but that is just my personal opinion don't really know what our prefered way is. We're using a render prop pattern here to propagate values that are fetched in the backend. This ReactProfileWrapper could instead be connected to redux state where data is already in place.